### DISCUSSION TITLE
What is the meaning of "ISODATA minimum class size in pixels"?

### DISCUSSION THREAD

**Discussion by FelipeSBarros** [semiautomaticgit/SemiAutomaticClassificationPlugin#260] [Q&A] [answered]
Hi @semiautomaticgit ! Congratulations for the amazing work done!
I have been working with ISODATA algorithm and would like to confirm the meaning of the parameter "minimum class size in pixels":

![image](https://user-images.githubusercontent.com/7208674/233462435-4938c9dd-2f2b-4d83-8db1-3d1c468052ac.png)

Does this parameter constrain the clusters' occurrence  to at least 10 pixels or it is a constrain to the class, with no matter of its spatial occurrence (like a class with only ten pixels spread in the image)?

**Comment by semiautomaticgit** [comment, marked as answer]
Hi @FelipeSBarros and thank you!
About "minimum class size in pixels" it is referred to the sum of class pixels in the raster, therefore also sparse pixels will count.
